Update more details-Observability service (in review) (#7974)
open-cluster-management/rhacm-docs: observability/observability_enable.adoc

== Prerequisites
 
- You must install {acm}. See link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online] for more information.

> - You must specify the `storageConfig.storageClass` field in the `MultiClusterObservability` custom resource if you do not want to use the platform's default `StorageClass`.

Reviewer: Removing possessive for inanimate object according to the IBM Style Guide: https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives
```suggestion
- You must specify the `storageConfig.storageClass` field in the `MultiClusterObservability` custom resource if you do not want to use the platform default `storageClass`.
```

+
[source,bash]
----

> $ oc create namespace open-cluster-management-observability

Reviewer 2: remove $

Reviewer 2: keep consistent throughout. This topic should have "required access:" at the top, to fit with the ACM format. -- just checked and it does.

Reviewer 2: This file also has ROKS in it as a product name, which is against company standards so that needs to change, as well.

> $ oc create namespace open-cluster-management-observability

Reviewer: ```suggestion
oc create namespace open-cluster-management-observability
```

+
[source,bash]
----

> $ oc create secret generic multiclusterhub-operator-pull-secret \

Reviewer: ```suggestion
oc create secret generic multiclusterhub-operator-pull-secret \
```

+
[source,bash]
----

> $ oc create -f thanos-object-storage.yaml -n open-cluster-management-observability

Reviewer: ```suggestion
oc create -f thanos-object-storage.yaml -n open-cluster-management-observability
```

+
[source,bash]
----

> $ oc apply -f multiclusterobservability_cr.yaml

Reviewer: ```suggestion
oc apply -f multiclusterobservability_cr.yaml
```

----

+

> *Note:* By default if you do not define the `storageConfig.storageClass` field in the `MultiClusterObservability` custom resource,  platforms default `StorageClass` will be populated in the   storageConfig  section of `MultiClusterObservability`. For example for AWS default storageClass will be gp2. You can verify default storageClass using below command:

Reviewer 2: ```suggestion
*Note:* By default, if you do not define the `storageConfig.storageClass` field in the `MultiClusterObservability` custom resource, platform default `StorageClass` fields are populated in the `storageConfig` section of the `MultiClusterObservability` resource. For example, AWS default `storageClass` is set to `gp2`. You can verify default `storageClass` by running the following command:
```

Reviewer 2: At ACM docs, we try to follow company and industry standards: present tense, coding non-English words that cannot be translated bc it is part of the code, and no bidirectional language like "below" that alienates a non-sighted user. See those changes.

> + 
> ----
> $ oc get storageClass
> NAME                PROVISIONER       RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE

Reviewer: ```suggestion
+
[source,bash] 
----
oc get storageClass
NAME                PROVISIONER       RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
```

gp2-csi             ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   151m
gp3-csi (default)   ebs.csi.aws.com   Delete          WaitForFirstConsumer   true                   151m
----

+

> All the pods in `open-cluster-management-observability` namespace for Thanos, Grafana and Alertmanager are created. All the managed clusters connected to the {acm-short} hub cluster are   enabled to send metrics back to the {acm-short} Observability service.

Reviewer 2: ```suggestion
All the pods in `open-cluster-management-observability` namespace for Thanos, Grafana, and Alertmanager are created. All the managed clusters connected to the {acm-short} hub cluster are enabled to send metrics back to the {acm-short} Observability service.
```

Reviewer 2: line 505 really should not be here--we don't know that they "are created";  for a while some writers were writing this in place of "verification steps" (this is not a verification step) without checking in for peer review--we have stopped doing that. I think this should be removed. You don't know that the pods are created.

> All the pods in `open-cluster-management-observability` namespace for Thanos, Grafana and Alertmanager are created. All the managed clusters connected to the {acm-short} hub cluster are   enabled to send metrics back to the {acm-short} Observability service.

Reviewer: ```suggestion
```

----
oc apply -f multiclusterobservability_cr.yaml
----

+

> *Note:* By default, if you do not define the `storageConfig.storageClass` field in the `MultiClusterObservability` custom resource, platform default `StorageClass` fields are populated in the `storageConfig` section of the `MultiClusterObservability` resource. For example, AWS default `storageClass` is set to `gp2`. You can verify default `storageClass` by running the following command:

Reviewer 2: There is a step inside this note.

+
[source,bash]
----

> $ oc get deploy multicluster-observability-operator -n open-cluster-management --show-labels

Reviewer: ```suggestion
oc get deploy multicluster-observability-operator -n open-cluster-management --show-labels
```
